### PR TITLE
Add overflow, transition, and transform properties.

### PR DESCRIPTION
**Pull request by josephdburdick**
This PR adds missing properties to the `declaration-block-properties-order` config rule.

### CODE REVIEW

**Comment by danilobuerger** [comment]
Thanks @josephdburdick for the PR. Please take a look at https://github.com/brandon-rhodes/Concentric-CSS/ which is the order I want to reflect here.

**Comment by josephdburdick** [comment]
Hi @danilobuerger are you saying you want all attributes listed from https://github.com/brandon-rhodes/Concentric-CSS/blob/master/style3.css ?

**Comment by danilobuerger** [comment]
I just want to keep the order the same. Specifically:
```
  /* transform */
  backface-visibility: ;
  perspective: ;
  perspective-origin: ; 
  transform: ;
  transform-origin: ;
  transform-style: ;

  /* tranistions */
  transition: ;
  transition-delay: ;
  transition-duration: ;
  transition-property: ;
  transition-timing-function: ;
```

**Comment by josephdburdick** [comment]
Hi @danilobuerger, if I make this change can I include *all* possible CSS attributes from `#Concentric-CSS-Complete` in that list?

**Comment by danilobuerger** [comment]
Sure, if you want to, go ahead!

**Comment by josephdburdick** [comment]
@danilobuerger All properties have been applied.

File: index.js [hunk 1]
```diff
@@ -105,35 +116,67 @@ config.rules['declaration-block-properties-order'] = [[
...
   'max-height',
   'overflow',
+  'overflow-x',
+  'overflow-y',
+  'resize',
   'list-style',
+  'list-style-type',
+  'list-style-position',
+  'list-style-image',
+  'caption-side',
```
Comment (danilobuerger) [inline index.js]: Duplicate, see next line

File: index.js [hunk 2]
```diff
@@ -105,35 +116,67 @@ config.rules['declaration-block-properties-order'] = [[
...
   'text-rendering',
   'text-shadow',
   'text-overflow',
   'line-height',
-  'word-break',
-  'word-wrap',
   'word-spacing',
   'letter-spacing',
   'white-space',
+  'word-break' ,
```
Comment (danilobuerger) [inline index.js]: Remove whitespace before `,`

File: index.js [hunk 3]
```diff
@@ -29,22 +30,25 @@ config.rules['declaration-block-properties-order'] = [[
...
+  'column-rule-style',
+  'column-rule-color',
   'column-span',
   'column-count',
   'column-width',
   'float',
   'clear',
+  'backface-visibility',
+  'perspective',
+  'perspective-origin' ,
```
Comment (danilobuerger) [inline index.js]: Remove whitespace before `,`

**Review by danilobuerger — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (josephdburdick) [reply index.js]: The `background-repeat` is not being repeated here, it's being removed from line 92 and repositioned to better match the array represented in the repo you linked me to.

Comment (josephdburdick) [reply index.js]: My mistake, I see the repeated value now.

**Review by danilobuerger — APPROVED** [review]

**Comment by danilobuerger** [comment]
@josephdburdick Thanks again for the great work, I really appreciate it. Would you mind squashing to one commit & amending, or should I do it for you?

**Comment by josephdburdick** [comment]
@danilobuerger Squashed down to 1 commit.

**Comment by danilobuerger** [comment]
Thanks!

**Pull request merged**